src/test/gcp/cloudfunctions.spec.ts

```diff
@@ -258,4 +258,50 @@ describe("cloudfunctions", () => {
       });
     });
   });
+
+  describe("generateIamPolicy", () => {
+    it("should throw error on empty service account string", () => {
+      expect(() => cloudfunctions.generateIamPolicy("project", "")).to.throw();
+    });
+
+    it("should generate the public policy", () => {
+      const policy = cloudfunctions.generateIamPolicy("project", "public");
```

Comment:
Its a good practice to split your tests with newlines into arrange, act, assert: go/unit-testing-practices#structure

For these, theres no arrange step, but you can throw a newline between the call to generateIamPolicy and the expects

Reply:
Cleaned up spacing in the tests